core-war/src/main/java/org/silverpeas/web/jobdomain/control/JobDomainPeasSessionController.java

```diff
@@ -1676,27 +1681,13 @@ public String createSQLDomain(String domainName, String domainDescription,
   public String modifyDomain(String domainName, String domainDescription,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Method has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00107)